addons/ui/ACE_UI.hpp

```diff
@@ -0,0 +1,177 @@
+// IDD, elements[]s, Element (must be string to compare to changed setting name), Selective Type (0-both, 1-ground, 2-vehicle)
+class ACE_UI {
+    class weaponName {
+        idd = 300;
+        elements[] = {118};
+        condition = QUOTE(ACE_player == vehicle ACE_player);
+        class conditions {};
```

Comment:
I had that at first, but this makes it easier to handle, because the main condition gets evaluated before any other and if not applicable it doesn't toggle the element at all (no point as you can't see it anyways), while other conditions determine if it should be visible or hidden.